src/convertToObject.js

```diff
@@ -5,8 +5,30 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const delEmptyLine = sourceString.replace(/\n/g, '');
+
+  const commandsArr = delEmptyLine.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `commandsArr`. It will decrease code readability. Consider using a more descriptive name, such as `styleDeclarations`.